common/services/prismic/parsers.js

```diff
@@ -544,6 +547,31 @@ export function parseOnThisPage(fragment: PrismicFragment[]): Link[] {
     });
 }
 
+export function parseMediaObjectList(
+  fragment: PrismicFragment[]
+): Array<MediaObjectType> {
+  // filter media-object
+  const filteredMediaObject = fragment.filter(mediaObject => {
+    return mediaObject.content?.type === 'media-object';
```

Comment:
Sorry to be annoying on this - but what is this doing? I thought we would only be able to have `media-object`s?

Reply:
no worries, basically the editors are able to save a linked document that is blank. wanted to make sure its filtered to this. I can remove this as i am checking now if there are missing data just give null.